test/e2e/network/dns.go

```diff
@@ -321,7 +321,7 @@ var _ = SIGDescribe("DNS", func() {
 		validateTargetedProbeOutput(f, pod3, []string{wheezyFileName, jessieFileName}, svc.Spec.ClusterIP)
 	})
 
-	It("should support configurable pod resolv.conf", func() {
+	It("should support configurable pod resolv.conf [LinuxOnly]", func() {
```

Comment:
Right, this test expects some particular content to be added to ``/etc/resolv.conf``. While the file does not exist on Windows, similar settings are being set in the container, so we could have a Windows equivalent.

Reply:
Thanks for reviewing @bclau. Included these as a comment.